CONTRIBUTING.md

```diff
@@ -67,3 +67,12 @@ Must be a list of measurable achievements that indicate the work in the issue is
 4. Don't just hope for the code to work, Check it yourself!
 5. Be strict about temporary code
 6. Visualize the bigger picture
+
```

Comment:
It is already part of:
https://github.com/redhat-beyond/commune/blob/main/README.md 
Please remove it

Reply:
@RonTuretzky please reply to the comments and rebase